sdk/azidentity/live_test.go

```diff
@@ -56,12 +57,42 @@ var liveUser = struct {
 }
 
 const (
-	fakeClientID   = "fake-client-id"
-	fakeResourceID = "/fake/resource/ID"
-	fakeTenantID   = "fake-tenant"
-	fakeUsername   = "fake@user"
+	fakeClientID      = "fake-client-id"
+	fakeResourceID    = "/fake/resource/ID"
+	fakeTenantID      = "fake-tenant"
+	fakeUsername      = "fake@user"
+	fakeAdfsAuthority = "fake.adfs.local"
+	fakeAdfsScope     = "fake.adfs.local/fake-scope/.default"
 )
 
+var adfsAuthority = os.Getenv("ADFS_AUTHORITY_HOST")
+
+var adfsLiveSP = struct {
+	tenantID    string
+	clientID    string
+	secret      string
+	pemPath     string
+	scope       string
+	redirectURL string
+}{
+	tenantID:    "adfs",
```

Comment:
Since this value won't change, I would just use it directly in all the tests and forego this field. If you really want the field, please also add it to `adfsLiveUser` so future readers don't wonder why user auth tests use service principal configuration.